applications/ContactStructuralMechanicsApplication/custom_processes/base_contact_search_process.cpp

```diff
@@ -744,7 +744,7 @@ void BaseContactSearchProcess<TDim, TNumNodes, TNumNodesMaster>::SearchUsingOcTr
                 // If not active we check if can be potentially in contact
                 if (mCheckGap == CheckGap::MappingCheck) {
                     for (auto p_leaf : leaves) {
-                        for (auto p_cond_master : *(p_leaf->pGetObjects())) {
+                        for (const auto& p_cond_master : *(p_leaf->pGetObjects())) {
```

Comment:
ok, however i would say that you were only making a copy of a pointer

Reply:
The reference would be always better